knowledge-base/system-null-reference-exception-docker.md

```diff
@@ -0,0 +1,39 @@
+---
+title: System.NullReferenceException: Object reference not set to an instance of an object
+description: Such error might be thrown in docker for Linux 
+type: troubleshooting
+page_title: System.NullReferenceException: Object reference not set to an instance of an object. at Telerik.Reporting.Drawing.FontExtensions.ToGdiFont(IFont font)
+slug: system- null-reference-exception-docker
+position: 
+tags: 
+ticketid: 1445482
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>13.0.19.116 +</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+sfsdf
```

Comment:
needs description, something like System.NullReferenceException: Object reference not set to an instance of an object when rendering report in a Docker container on Linux